test/e2e/provisioning/helper.go

```diff
@@ -33,9 +33,11 @@ var (
 	scenarios = buildScenarios()
 
 	versions = []*semver.Version{
-		semver.MustParse("v1.15.6"),
-		semver.MustParse("v1.16.3"),
-		semver.MustParse("v1.17.0"),
+		semver.MustParse("v1.15.11"),
```

Comment:
We should consider dropping 1.15 support because it has reached EOL several months ago.

Reply:
indeed